lib/v2/xiaohongshu/util.js

```diff
@@ -68,7 +69,133 @@ const getBoard = (url, cache) =>
+                        )
+                    );
+                } catch (e) {
+                    throw `Could not get user information and note list\n${e}`;
+                }
+
+                // Get full text for each note
+                const notesPromise = userPosted.data.notes.map((n) => {
+                    const noteUrl = url + '/' + n.note_id;
+
+                    return cache.tryGet(
+                        noteUrl,
+                        async () => {
+                            const notePage = await browser.newPage();
+                            await setPageFilter(notePage);
+
+                            logger.debug(`Requesting ${noteUrl}`);
+                            await notePage.goto(noteUrl);
+
+                            let feed = {};
+                            try {
+                                feed = await notePage.evaluate(() => window.__INITIAL_STATE__);
+
+                                // Sometimes the page is not server-side rendered
+                                if (typeof feed?.note?.note === 'undefined' || JSON.stringify(feed?.note?.note) === '{}') {
+                                    const res = await notePage.waitForResponse((res) => {
+                                        const req = res.request();
+                                        return req.url().includes('/api/sns/web/v1/feed') && req.method() === 'POST';
+                                    });
+
+                                    const json = await res.json();
+                                    const note_card = json.data.items[0].note_card;
+                                    feed.note.note = {
+                                        title: note_card.title,
+                                        noteId: note_card.id,
+                                        desc: note_card.desc,
+                                        tagList: note_card.tag_list,
+                                        imageList: note_card.image_list,
+                                        user: note_card.user,
+                                        time: note_card.time,
+                                        lastUpdateTime: note_card.last_update_time,
+                                    };
+                                }
+                            } catch (e) {
+                                throw `Could not get note ${n.note_id}\n${e}`;
+                            }
+
+                            await notePage.close();
+
+                            if (typeof feed?.note?.note !== 'undefined' && JSON.stringify(feed?.note?.note) !== '{}') {
+                                return feed.note.note;
+                            } else {
+                                throw 'Could not get note ${n.note_id}';
+                            }
+                        },
+                        config.cache.routeExpire * 4, // Use a larger expire time for note full text
+                        true
+                    );
```

Comment:
```suggestion
                        }
                    );
```
Skipping the 3rd and the 4th parameter will use the longer cache time `config.cache.contentExpire` and `refresh=true`